src/pages/SharePage/index.tsx

```diff
@@ -9,8 +10,18 @@ import {
 } from "../../components";
 import "./SharePage.css";
 
+interface FolderInfos {
+  owner: {
+    profileImageSource: string;
+    name: string;
+  };
+  name: string;
+  links: Link[];
+}
+
 const SharePage = () => {
-  const [folderData, setFolderData] = useState([]);
+  const [folderData, setFolderData] = useState<FolderInfos>({} as FolderInfos);
+  const [searchTerm, setSearchTerm] = useState<string>("");
 
   const handleLoad = async () => {
     const { folder } = await getFolder();
```

Comment:
`getFolder`의 리턴 타입이 `any`네요. 내부 구현을 보니까 `fetch`와 아무런 타입지정이 없습니다.
`fetch`에 타입을 줄 수 있는 곳을 딱히 보이지 않네요. 이럴때 `as`를 써주면 됩니다. 

특히 api로 받는 부분은 타입을 반드시 지정해서 any 타입의 데이터가 프론트 코드로 흘러들어가지 않게 막아주세요.

```suggestion
type FolderResponse = {
    folder: FolderInfos;
};

const response = await fetch(`${URL}/sample/folder`);
        const body = (await response.json()) as FolderResponse;
```